### ISSUE TITLE
Problem with installation FerriteAssembly.jl

### ISSUE THREAD

**Issue by bplcn** [KnutAM/FerriteAssembly.jl#25]
When I install the package and compile it,  it reports that 

```julia Repl
(FerriteAssembly_test) pkg> precompile
Precompiling project...
  ✗ FerriteAssembly
  0 dependencies successfully precompiled in 49 seconds. 44 already precompiled.

ERROR: The following 1 direct dependency failed to precompile:

FerriteAssembly [fd21fc07-c509-4fe1-9468-19963fd5935d]

Failed to precompile FerriteAssembly [fd21fc07-c509-4fe1-9468-19963fd5935d] to "C:\\Users\\HP\\.julia\\compiled\\v1.9\\FerriteAssembly\\jl_5C41.tmp".
ERROR: LoadError: UndefVarError: `SubDofHandler` not defined
Stacktrace:
 [1] top-level scope
   @ C:\Users\HP\.julia\packages\FerriteAssembly\JN3vu\src\Utils\FerriteAddons.jl:4
 [2] include(mod::Module, _path::String)
   @ Base .\Base.jl:457
 [3] include(x::String)
   @ FerriteAssembly C:\Users\HP\.julia\packages\FerriteAssembly\JN3vu\src\FerriteAssembly.jl:1
 [4] top-level scope
   @ C:\Users\HP\.julia\packages\FerriteAssembly\JN3vu\src\FerriteAssembly.jl:7
 [5] include
   @ .\Base.jl:457 [inlined]
 [6] include_package_for_output(pkg::Base.PkgId, input::String, depot_path::Vector{String}, dl_load_path::Vector{String}, load_path::Vector{String}, concrete_deps::Vector{Pair{Base.PkgId, UInt128}}, source::Nothing)
   @ Base .\loading.jl:2010
 [7] top-level scope
   @ stdin:2
in expression starting at C:\Users\HP\.julia\packages\FerriteAssembly\JN3vu\src\Utils\FerriteAddons.jl:4
in expression starting at C:\Users\HP\.julia\packages\FerriteAssembly\JN3vu\src\FerriteAssembly.jl:1
in expression starting at stdin:2

(FerriteAssembly_test) pkg> rm FerriteAssembly
    Updating `D:\JianguoYun_Files\OthersPackage\FerriteAssembly_test\Project.toml`
  [fd21fc07] - FerriteAssembly v0.3.3 `https://github.com/KnutAM/FerriteAssembly.jl#main`
    Updating `D:\JianguoYun_Files\OthersPackage\FerriteAssembly_test\Manifest.toml`
  [944b1d66] - CodecZlib v0.7.3
  [b4f34e82] - Distances v0.10.11
  [4e289a0a] - EnumX v1.0.4
  [c061ca5d] - Ferrite v0.3.14
  [fd21fc07] - FerriteAssembly v0.3.3 `https://github.com/KnutAM/FerriteAssembly.jl#main`
  [1a297f60] - FillArrays v1.9.3
  [9c8b4983] - LightXML v0.9.1
  [b8a86587] - NearestNeighbors v0.4.16
  [189a3867] - Reexport v1.2.2
  [82ae8749] - StatsAPI v1.7.0
  [3bb67fe8] - TranscodingStreams v0.9.13
  [4004b06d] - VTKBase v1.0.1
  [64499a7a] - WriteVTK v1.18.1
  [94ce4f54] - Libiconv_jll v1.17.0+0
  [02c8fc9c] - XML2_jll v2.12.2+0
```

Is there a problem of the dependency information of Ferrite? Thank you for the help.

**Comment by KnutAM** [comment]
Yes, currently, it is only compatible with Ferrite.jl master. 
So it should work if you do `pkg> add Ferrite#master`.
(I thought we would have finished releasing Ferrite v1.0 some time ago, but unfortunately not done yet...)

**Comment by bplcn** [comment]
> add Ferrite#master

It works! Thank you. By the way, the MaterialModelBase.jl is required when installation
```Julia repl
(FerriteAssembly_test) pkg> add https://github.com/knutam/FerriteAssembly.jl
    Updating git-repo `https://github.com/knutam/FerriteAssembly.jl`
   Resolving package versions...
ERROR: Unsatisfiable requirements detected for package MaterialModelsBase [af893363]:
 MaterialModelsBase [af893363] log:
 ├─MaterialModelsBase [af893363] has no known versions!
 └─restricted to versions 0.2 by FerriteAssembly [fd21fc07] — no versions left
   └─FerriteAssembly [fd21fc07] log:
     ├─possible versions are: 0.3.3 or uninstalled
     └─FerriteAssembly [fd21fc07] is fixed to version 0.3.3
```
I installed it manually and had no problem when running. I am wondering if it is also a problem with the dependencies.

**Comment by KnutAM** [comment]
Thanks for the comments - I've added instructions to the readme. 
Unfortunately, `MaterialModelsBase.jl` has to be handled manually until https://github.com/JuliaLang/Pkg.jl/issues/492 is resolved.
